src/test/java/net/snowflake/client/jdbc/ResultSetIT.java

```diff
@@ -1056,4 +1056,19 @@ public void testNextNegative() throws SQLException {
       }
     }
   }
+
+  /** SNOW-1416051; Added in > 3.16.0 */
+  @Test
+  public void shouldSerializeArrayAsObject() throws SQLException {
+    try (Connection connection = init();
+        Statement statement = connection.createStatement(); ) {
```

Comment:
Semicolon to delete?

Reply:
deleted

Reply:
changed the structure to have only one try